cli/config/configuration.toml.in

```diff
@@ -211,8 +211,11 @@ path = "@SHIMPATH@"
 #enable_debug = true
 
 [agent.@PROJECT_TYPE@]
-# There is no field for this section. The goal is only to be able to
-# specify which type of agent the user wants to use.
+# If enabled, when we pass a block device to the guest VM
+# through -v, such as `docker run -v /dev/loop100:/foo/bar`,
+# agent will create a directory in guest VM and mount the
+# file system on `/dev/loop100` inside the container
+#enable_blk_mount_in_vm = true
```

Comment:
I am afraid we have already bloated the number of things that can be configured in our configuration.toml and shuld consider what we add here.
What if we have this configured as an OCI annotation instead, that we we can configure it for every pod instead of having this as a host wide configuration. 

What do others think about this  cc @kata-containers/runtime